docker/base/Dockerfile

```diff
@@ -1,9 +1,9 @@
 ARG cert_image
 ARG root_image
 
-FROM $cert_image AS cert
+FROM alpine:3.19 AS cert
```

Comment:
what was the reason a variable was used?

Reply:
since the the CERT_IMAGE is based on the ROOT_IMAGE in the makefile, shouldn't we change it here too

Reply:
here is an example of overriding the variable
```
./scripts/build-upload-a-docker-image.sh:19:		base_debug_img_arg="--build-arg base_image=localhost:5000/baseimg_alpine:latest --build-arg debug_image=localhost:5000/debugimg_alpine:latest "
```

You cannot just replace it with a constant

Reply:
I would remove all 3 variables that define versions from the Makefile (and places where they are passed in) and just give default values to the vars